File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {

        for(int i = 0; i < nums.Length; i++){
            for(int j = 0; j < nums.Length; j++){
                if(nums[j] == nums[i] && i != j){
                    return true;
                }
            }
        }
        return false;
    }
    
}

File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int res = 0;
        if(nums.length == 1){
            return nums[0];
        }

        for(int i = 0; i < nums.length; i++){
            res += nums[i];
            if(res < nums[i]){
                res = nums[i];
            }
            // System.out.println(res + " number was " + nums[i]);
        }

        return res;
    }
}
